docs/book/v1/authorization-server.md

```diff
@@ -68,46 +68,46 @@ use Psr\Http\Message\ServerRequestInterface;
 use Psr\Http\Server\RequestHandlerInterface;
 use Psr\Http\Message\ResponseInterface;
 use League\OAuth2\Server\RequestTypes\AuthorizationRequest;
+use Zend\Expressive\Authentication\UserInterface;
 
 class OAuthAuthorizationMiddleware implements MiddlewareInterface
 {
-    public function process(ServerRequestInterface $request, RequestHandlerInterface $handler): ResponseInterface 
+    public function process(ServerRequestInterface $request, RequestHandlerInterface $handler): ResponseInterface
     {
         // Assume a middleware handled the authentication check and
         // populates the user object which also implements the
         // OAuth2 UserEntityInterface
-        $user = $request->getAttribute('authenticated_user');
-        
+        $user = $request->getAttribute(UserInterface::class);
```

Comment:
We need to document this change in the `CHANGELOG.md` as well.

Reply:
This was actually a documentation error introduced by #42. The correct attribute for user was `UserInterface::class` and not `authenticated_user`.